client/components/tinymce/plugins/wpcom-view/views/video/view.jsx

```diff
@@ -35,7 +35,9 @@ function VideoView( { siteId, content, width } ) {
 		)
 	);
 
-	return <Shortcode { ...{ siteId, width } }>{ shortcode }</Shortcode>;
+	const allowSameOrigin = true;
+
+	return <Shortcode { ...{ siteId, width, allowSameOrigin } }>{ shortcode }</Shortcode>;
```

Comment:
I know this was already there, but I think this would be simpler if we just passed all these props directly. No real reason to use expansion when we know all the props and we have to create an object to do it.

IE, I'd prefer `<Shortcode siteId={ siteId } width={ width } allowSameOrigin={ true }>...`

Reply:
That would also remove the need for the new `const`